packages/terra-heading/tests/nightwatch/HeadingSize.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+import Heading from '../../lib/Heading';
+
+const HeadingSize = () => (
+  <div>
```

Comment:
I think that would exist better in the doc site in my opinion. I think the nightwatch tests should be as isolated to a prop as possible, as adding more future headings and styles could make this page rather large.